src/store/reducers/storage.js

```diff
@@ -6,8 +6,8 @@ import {calcUptime} from '../../utils';
 
 export const VisibleEntities = {
     All: 'All',
-    Missing: 'Missing',
-    Space: 'Space',
+    Missing: 'Degraded',
+    Space: 'Out of Space ',
```

Comment:
'Out of space'

Do you actually need trailing space?

Reply:
No, that's typo!